cookbook/caching-with-varnish.rst

```diff
@@ -148,6 +148,33 @@ The following will add full caching support for Sulu:
     sub vcl_recv {
         call sulu_recv;
 
+        // # Add a Surrogate-Capability header to announce ESI support.
+        // set req.http.Surrogate-Capability = "abc=ESI/1.0";
+
+        // # Remove all cookies except the session ID (SULUSESSID)
+        // # Check configured session ID name in your config/packages/framework.yaml
+        // if (req.http.Cookie) {
+        //     set req.http.Cookie = ";" + req.http.Cookie;
+        //     set req.http.Cookie = regsuball(req.http.Cookie, "; +", ";");
+        //     set req.http.Cookie = regsuball(req.http.Cookie, ";(SULUSESSID)=", "; \1=");
+        //     set req.http.Cookie = regsuball(req.http.Cookie, ";[^ ][^;]*", "");
+        //     set req.http.Cookie = regsuball(req.http.Cookie, "^[; ]+|[; ]+$", "");
+
+        //     if (req.http.Cookie == "") {
+        //         # If there are no more cookies, remove the header to get page cached.
+        //         unset req.http.Cookie;
+        //     }
+        // }
+
+        // if (req.method != "GET" && req.method != "HEAD") {
+        //     return (pass);
+        // }
+
+        // # In different to the builtin.vcl of varnish we cache the page still when it have a cookie
```

Comment:
i think this comment is missleading - the following statement says that the request will not be cached when a Authorization header isset.

Reply:
Ok that make we know think that this should not be the case but I'm not so familiar with varnish /cc @chirimoya is this config false or should that be the case?

Reply:
fixed this based on our projects configuration